lib/common/actions/index.js

```diff
@@ -42,43 +49,61 @@ export function secureFetch (url: string, method: string = 'get', payload?: any,
     return fetch(url, {method, headers, body})
       // Catch basic error during fetch
       .catch(err => {
-        const message = `Error making request: (${err})`
-        consoleError(message)
+        const message = `Error making ${method} request to ${url}`
```

Comment:
Extract a var for this message.

Reply:
Added var in https://github.com/ibi-group/datatools-ui/pull/694/commits/7242f13da7192bc93fb1d537532a92cb68869981